pkgs/applications/misc/goldendict-ng/default.nix

```diff
@@ -0,0 +1,83 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, cmake
+, libvorbis
+, ffmpeg
+, libeb
+, hunspell
+, opencc
+, xapian
+, libzim
+, lzo
+, xz
+, tomlplusplus
+, fmt
+, bzip2
+, libiconv
+, libXtst
+, qtbase
+, qtsvg
+, qtwebengine
+, qttools
+, qtwayland
+, qt5compat
+, qtmultimedia
+, qtspeech
+, wrapQtAppsHook
+}:
+
+stdenv.mkDerivation rec {
+  pname = "goldendict-ng";
+  version = "23.07.23";
+
+  src = fetchFromGitHub {
+    owner = "xiaoyifang";
+    repo = pname;
```

Comment:
Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21